tap_chargebee/streams/base.py

```diff
@@ -128,7 +159,29 @@ def sync_data(self):
                 method=api_method,
                 params=params)
 
-            to_write = self.get_stream_data(response.get('list'))
+            records = response.get('list')
+            
+            to_write = self.get_stream_data(records)
+            
+            if self.ENTITY == 'event':
+                for event in to_write:
+                    if event["event_type"] == 'plan_deleted':
+                        Util.plans.append(event['content']['plan'])
+                    elif event['event_type'] == 'addon_deleted':
+                        Util.addons.append(event['content']['addon'])
+                    elif event['event_type'] == 'coupon_deleted':
+                        Util.coupons.append(event['content']['coupon'])
+            if self.ENTITY == 'plan':
+                for plan in Util.plans:
+                    to_write.append(plan)
+            if self.ENTITY == 'addon':
+                for addon in Util.addons:
+                    to_write.append(addon)
+            if self.ENTITY == 'coupon':
+                for coupon in Util.coupons:
+                    to_write.append(coupon) 
```

Comment:
Was this included intentionally? Seems to not be related to the rest of the PR.

Reply:
It seems while merging it is added. Now removed.